### PR TITLE
Add shapes example 5

### PR DESCRIPTION
**Pull request by mjkalyan**
Ready to merge.

### CODE REVIEW

**Review by shelvick — CHANGES_REQUESTED** [review]
AFAICT you are only using `#:str` for `substring`? If that's the case, you don't need it. Use the `subseq` built-in instead.

File: examples/shapes/05-logo-raylib-anim.lisp [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                         alpha 1.0
+                         state 0))))
+
+            ;; Change the colour fading
+            (fade *faded-black* alpha)
+            (fade *faded-white* alpha)
+
+            (with-drawing
+              (case state
+                (0 (when (= 0 (mod (truncate frames-counter 15) 2))
```
Comment (shelvick) [inline examples/shapes/05-logo-raylib-anim.lisp:90]: Also, does this not work via `draw-scene`?

Comment (shelvick) [reply examples/shapes/05-logo-raylib-anim.lisp:90]: Errr... the lines below this, not above it. GitHub confuses me sometimes.

Comment (mjkalyan) [reply examples/shapes/05-logo-raylib-anim.lisp:90]: Yeah `draw-scene` totally works. I it was nice to have a plural for `draw-object` (handy when an object is already in scope). If you don't like that, I'll scrap it. If you do like it, we could add it to claylib itself.

Comment (shelvick) [reply examples/shapes/05-logo-raylib-anim.lisp:90]: Yeah, `draw-objects` is fine. I forgot we're basically already doing that anyway, just with a scene. You can add that into Claylib proper and resubmit.

Comment (mjkalyan) [reply examples/shapes/05-logo-raylib-anim.lisp:85]: Done (https://github.com/defun-games/claylib/pull/21/commits/1634cb88b27ed1147367e772f3947424d58cd2c5)! Should be good to merge.

**Pull request merged**